README.md

```diff
@@ -263,6 +263,14 @@ FactoryHelper::Name.prefix #=> "Mr."
 FactoryHelper::Name.suffix #=> "IV"
 
 FactoryHelper::Name.title #=> "Legacy Creative Director"
+
+# NOTE NOTE
+# For the 'US only' methods below, first you must do the following:
+FactoryHelper::Config.locale = 'en-US'
+
+# US only
+FactoryHelper::Name.girls_name #=> "Marissa"
```

Comment:
What about scoping this under `en`? Even though it has US data, it should be broadly applicable to all English-speaking locales. In the future, if we have data for UK, CA, AUS etc., we can narrow the scopes again.